modules/mongodb/scanner.go

```diff
@@ -64,6 +65,17 @@ func getBuildInfoQuery() []byte {
 	return query_msg
 }
 
+// getListDatabasesMsg returns a mongodb message containing a command to retrieve MongoDB database info.
+func getListDatabasesMsg() []byte {
+	query, err := bson.Marshal(bson.M{"listDatabases": 1})
+	if err != nil {
+		// programmer error
+		log.Fatalf("Invalid BSON: %v", err)
+	}
+	query_msg := getOpQuery("admin.$cmd", query)
```

Comment:
go naming way it use `camelCase` -> `queryMsg`

Reply:
The naming here follows the convention used elsewhere in the file. I can update those, but it may be best to instead separate coding standards compliance into a separate PR alongside some enforcement built within the git pipeline.

Reply:
Okay, I agree with you